skycultures/greek_leidenAratea/description.en.utf8

```diff
@@ -0,0 +1,319 @@
+		<td> <img width="255" src="descr_pics/aratea_small_dra.jpg"/> </td>
+	</tr>
+	<tr valign="top">
+		<td> King Cepheus, Cep</td>
+		<td>&Kappa;&eta;&phi;&epsilon;&#973;&sigmaf; </td>
+		<td> Kepheus</td>
+		<td> <img width="255" src="descr_pics/aratea_small_cep.jpg"/> </td>
+	</tr>
+	<tr valign="top">
+		<td>Bootes Boo	</td>
+		<td> &Beta;&omicron;&#974;&tau;&eta;&sigmaf; </td>
+		<td>Ox driver (Latin), Ploughing Man (Greek) </td>
+		<td> <img width="255" src="descr_pics/aratea_small_boo.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Northern Crown, CrB 	</td>
+		<td>&Sigma;&tau;&#941;&phi;&alpha;&nu;&omicron;&sigmaf; </td>
+		<td>Crown (Latin), Wreath (Greek)</td>
+		<td> <img width="255" src="descr_pics/aratea_small_crB.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Hercules, Her</td>
+		<td>&#700;&Epsilon;&nu;&gamma;&#972;&nu;&alpha;&sigma;&iota;&nu;</td>
+		<td>Hercules (Latin), Kneeler (Greek) <br> The star between the legs is a well-known mistake in the picture: for this figure the draughtsman obviously took Bootes as a model; leg position, shepherd's crook (instead of club for Hercules) and star between the knees are the same.</td>
+		<td> <img width="255" src="descr_pics/aratea_small_her.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Lyra, Lyr</td>
+		<td>&Lambda;&#973;&rho;&alpha; </td>
+		<td>Lyre</td>
+		<td> <img width="255" src="descr_pics/aratea_small_lyr.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Cygnus, Cyg </td>
+		<td>&#700;&#714;&Omicron;&rho;&nu;&iota;&sigmaf; 	</td>
+		<td>Bird [not specified] </td>
+		<td> <img width="255" src="descr_pics/aratea_small_cyg.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Queen Cassiopeia, Cas</td>
+		<td>&Kappa;&alpha;&sigma;&sigma;&iota;&#941;&pi;&epsilon;&iota;&alpha; </td>
+		<td>Queen Kassiepeia </td>
+		<td> <img width="255" src="descr_pics/aratea_small_cas.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Perseus, Per	</td>
+		<td>&Pi;&epsilon;&rho;&sigma;&epsilon;&#973;&sigmaf;</td>
+		<td>Perseus <br> mirrored depiction</td>
+		<td> <img width="255" src="descr_pics/aratea_small_per.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Auriga, Aur		</td>
+		<td>&#699;&Eta;&nu;&#943;&omicron;&chi;&omicron;&sigmaf; </td>
+		<td>Charioteer	</td>
+		<td> <img width="255" src="descr_pics/aratea_small_aur.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Aquila et Sagitta,<br> Aql and Sge</td>
+		<td> &#702;&Alpha;&epsilon;&tau;&#972;&sigmaf; and &#702;&Omicron;&iota;&sigma;&tau;&#972;&sigmaf; </td>
+		<td> Eagle with Arrow </td>
+		<td> <img width="255" src="descr_pics/aratea_small_aql.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Delphinus, Del </td>
+		<td> &Delta;&epsilon;&lambda;&phi;&#943;&sigmaf; </td>
+		<td> Dolphin 	</td>
+		<td> <img width="255" src="descr_pics/aratea_small_del.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Pegasus, Peg</td>
+		<td> &#699;&#714;&Iota;&pi;&pi;&omicron;&sigmaf; </td>
```

Comment:
Accent &#699; should be &#703; (spiritus asper). Probably also elsewhere. It looks good for Aqr.

Reply:
danke, das wollte ich schon tausendmal machen (seit alm-SC)